core/src/components/popover/utils.ts

```diff
@@ -314,6 +314,11 @@ export const getPrevItem = (items: HTMLIonItemElement[], currentItem: HTMLElemen
   return items[currentItemIndex - 1];
 }
 
+/** Focus the internal button of the ion-item */
+const focusItem = (item: HTMLIonItemElement) => {
+  (item.shadowRoot?.querySelector('button.item-native') as HTMLElement)?.focus()
```

Comment:
```suggestion
  const root = getElementRoot(item);
  const button = root.querySelector('button');
  
  if (button) {
    button.focus();
  }
```

Can we use the `getElementRoot` function here instead? https://github.com/ionic-team/ionic-framework/blob/7315e0157b917d2e3586c64f8082026827812943/core/src/utils/helpers.ts#L97